### ISSUE TITLE
Enhancement: Ability to expand sky and ground radius

### ISSUE THREAD

**Issue by dsinni** [supermedium/aframe-environment-component#45]
It would be great if there were a simple way to expand the radius of the sky/starfield and the ground for more expansive immersion.

I mentioned the starfield, because you can see the stars "move" as the user moves. Allowing them to be farther from the camera would diminish this effect.

**Comment by feiss** [comment]
yup, interesting issue the one with stars.. Adding a parameter `size` would be nice indeed, I'll try to do it. My only concern is that expanding too much the ground while trying to preserve the same amount of detail could lend to a very dense geometry.. should I add another parameter `groundDetail`?

**Comment by dsinni** [comment]
I think that would be great, actually. It would leave it up to the discretion of the experience designer. I'm assuming `groundDetail` would be a radius?

In the case of the stars, specifically, I think it would matter less since the ground doesn't need to even remotely match the radius of a star field. I think for that, it would be ideal if the radius/distance could be significantly larger/farther, or at least have the option. I suppose they're separate issues but could be pretty closely related, hence the one issue. Let me know if you think I should create another.

Thanks for the reply, man.
